.vscode/settings.json

```diff
@@ -12,5 +12,5 @@
         "Include/": true,
         "*.egg-info/": true,
     },
-    "python.pythonPath": "bin/python"
+    "python.pythonPath": "/usr/bin/python3"
```

Comment:
Can you revert this? It's set this way to use a virtual environment (alternatively it can be `venv/bin/python`).